packages/lib/package.json

```diff
@@ -3,8 +3,10 @@
   "version": "2.1.4",
   "description": "leafyGreen UI Kit Internal Library",
   "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
   "scripts": {
-    "build": "../../node_modules/.bin/webpack --config ../../webpack.config.js"
+    "build": "../../node_modules/.bin/webpack --config ../../webpack.config.js",
+    "emit-types": "tsc --project ../../tsconfig.json --declaration --declarationDir dist --emitDeclarationOnly"
```

Comment:
Can these args just go into the tsconfig.json file? So we don't have to duplicate them across projects?

Reply:
That's probably a good idea – what are your thoughts on extending the base configuration in a new definition file-specific tsconfig for this?

Reply:
Oh hell yeah that sounds great. Maybe not duplicated across every individual package, but one other package-tsconfig.json that can be solely referenced.

Reply:
Took it as far as I could for the time being. Less duplication when we add this to other packages.